pkg/mei/ioctl_linux.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mei
+
+// this file contains a few ioctl-related constants and reimplementations of
+// macros from Linux's include/uapi/asm-generic/ioctl.h .
+
+const (
```

Comment:
Maybe we should have a separate ioctl package to avoid duplicating this:
https://github.com/u-root/u-root/blob/master/pkg/ipmi/ioctl_unix.go

Reply:
yes, that's what @hugelgupf suggested. I can fall back to the original `ioctl` package as discussed in https://github.com/u-root/u-root/pull/1642#discussion_r411036949 , and in another PR migrate `ipmi` to it

Reply:
done